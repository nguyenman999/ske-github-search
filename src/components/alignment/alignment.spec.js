import React from 'react'
import renderer from 'react-test-renderer'
import Alignment from './alignment'

describe('View: Alignment', () => {
  it('Renders', () => {
    const tree = renderer
      .create(
        <Alignment className="foo">
          <Alignment.Left>left</Alignment.Left>
          <Alignment.Center className="fee">center with className</Alignment.Center>
          <Alignment.Right>right</Alignment.Right>
        </Alignment>)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
