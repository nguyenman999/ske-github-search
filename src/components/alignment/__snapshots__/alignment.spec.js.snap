// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View: Alignment Renders 1`] = `
<div
  className="container foo"
>
  <div
    className="left"
  >
    left
  </div>
  <div
    className="center fee"
  >
    center with className
  </div>
  <div
    className="right"
  >
    right
  </div>
</div>
`;
